### DISCUSSION TITLE
How to use the root-tld option?

### DISCUSSION THREAD

**Discussion by bluesm866** [projectdiscovery/interactsh#171] [Q&A] [answered]
Version 0.0.7
I am running my server with the root-tld option.
How should I run the client to listen to a custom sub domain DNS interactions?

```
Server: 
./interactsh-server -domain mydomain -ip x.x.x.x -root-tld -token mytoken

Client:
./interactsh-client -server http://mydomain -token mytoken

```
If I ping test.mydomain there is no DNS interaction. But if I visit http://test.mydomain it will show me http requests to http://test.mydomain.

Only by pinging the subdomain created by the client I get DNS interaction.

**Comment by ehsandeep** [comment, marked as answer]
@bluesm866 might be related to https://github.com/projectdiscovery/interactsh/issues/141, it is something we fixed in the dev branch if you wanted to try before we push new release, also `-root-tld` is now updated to `-wildcard` flag in dev branch.
